astropy/io/votable/tests/test_dataorigin.py

```diff
@@ -0,0 +1,77 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import astropy.io.votable
+import astropy.io.votable.dataorigin as dataorigin
+from astropy.table import Column, Table
+
+
+def __generate_votable_test():
```

Comment:
In addition to the programmatic construction of a VOTable here, it would be nice to have at least one actual VOTable added to the data area to be parsed and tested here.  This would do a deeper tests than what is currently happening in the doc tests.